lib/Transforms/IPO/StructFieldAccessInfo.cpp

```diff
@@ -29,15 +29,27 @@
 
 using namespace llvm;
 
-static cl::opt<unsigned> HistogramSizeForStats(
-    "struct-analysis-number-buckets", cl::init(10), cl::Hidden,
-    cl::desc("Number of buckets used to analyze struct hotness"));
+static cl::opt<unsigned>
+    HistogramSizeForStats("struct-analysis-number-buckets", cl::init(10),
+                          cl::Hidden,
+                          cl::desc("Number of buckets used in the histogram "
+                                   "for analyzing struct hotness"));
+
+static cl::opt<bool> FilterOutZeroHotness(
+    "struct-analysis-filter-zero-hotness", cl::init(false), cl::Hidden,
```

Comment:
But that involves changing a few unit tests that doesn't profiling. I can add flags in the unit tests.